server/src/main/java/com/linecorp/centraldogma/server/metadata/Token.java

```diff
@@ -80,22 +85,26 @@ public Token(@JsonProperty("appId") String appId,
                  // TODO(minwoox): Remove admin field after all tokens are migrated.
                  @JsonProperty("admin") @Nullable Boolean isAdmin,
                  @JsonProperty("systemAdmin") @Nullable Boolean isSystemAdmin,
+                 @JsonProperty("allowGuestAccess") @Nullable Boolean allowGuestAccess,
                  @JsonProperty("creation") UserAndTimestamp creation,
                  @JsonProperty("deactivation") @Nullable UserAndTimestamp deactivation,
                  @JsonProperty("deletion") @Nullable UserAndTimestamp deletion) {
         assert isAdmin != null || isSystemAdmin != null;
         this.appId = Util.validateFileName(appId, "appId");
         this.secret = Util.validateFileName(secret, "secret");
         this.isSystemAdmin = isSystemAdmin != null ? isSystemAdmin : isAdmin;
+        // Allow guest access by default for backward compatibility.
```

Comment:
Note: Although `allowGuestAccess=true` is allowed via API, it isn't allowed via UI. I understood the reason is this field will eventually be deprecated and guest access won't be allowed entirely.

Reply:
Right. Users can't set `allowGuestAccess=true` either in the UI or the REST API.
`allowGuestAccess` is true for old tokens or administrative tokens.